python/test/t_FiniteOrthogonalFunctionFactory_std.py

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+
+ot.TESTPREAMBLE()
+
+# Check the parameter constructor
```

Comment:
Can you explain, in the comments of this script, how you were able to find this particular set of functions? This may help to maintain the test.